### PR TITLE
Global Times

### PR DESCRIPTION
**Pull request by sirpengi**
Fixes: #354 

Screenshot_20240205_162052

### CODE REVIEW

File: lib/widgets/content.dart [hunk 1]
```diff
@@ -687,6 +692,41 @@ class UserMention extends StatelessWidget {
 //     borderRadius: BorderRadius.all(Radius.circular(3))));
 }
 
+class GlobalTime extends StatelessWidget {
```
Comment (gnprice) [inline lib/widgets/content.dart:695]: nit: put this after MessageImageEmoji, to match the if/else-if chain above (which matches the node-class definitions)

File: test/widgets/content_test.dart [hunk 2]
```diff
@@ -93,6 +93,18 @@ void main() {
     tester.widget(find.text(r'\lambda'));
   });
 
+  testWidgets('GlobalTime smoke', (tester) async {
```
Comment (gnprice) [inline test/widgets/content_test.dart:96]: nit: put this after the MathInlineNode test below, to match the code under test in `lib/widgets/content.dart`

**Review by gnprice — COMMENTED** [review]
Thanks @sirpengi! Generally this looks good; just small comments below.

**Comment by sirpengi** [comment]
@gnprice ready for review again!

**Pull request merged**